docs/scripts/gen-faq.js

```diff
@@ -0,0 +1,151 @@
+// read.js
+const fs = require('fs')
+const yaml = require('js-yaml')
+const { Remarkable } = require('remarkable')
+const path = require('path')
+const yamlPath = path.resolve('./docs/faq.yaml')
+
+// Generating FAQ.mdx
+
+if (!fs.existsSync(yamlPath)) {
+  //file exists
+  console.warn('.yaml File does not exists, skipping generating FAQ')
+  return 0
+}
+
+let faqYaml = fs.readFileSync(yamlPath, 'utf8')
+let faq = yaml.load(faqYaml)
+
+const tags = Array.from(new Set(faq.map(({ tags }) => tags).flat(1)))
+
+let data = `---
+id: faq
+title: Frequently Asked Questions (FAQ)
+---
+
+
+
+import {Question, Faq} from '@theme/Faq'
+
+<Faq tags="${tags.join(' ')}"/>
```

Comment:
```suggestion
<Faq tags={${JSON.stringify(tags)}}/>
```

Reply:
Not sure what the advantage would be. In the case of stringify, we would pass an array json through a tags-atribute. Maybe we can then avoid the "split" at the other side, but also we need to take care of which kind of quotes to use and so on.

It feels much more natural to do it space-separated as that's the way you seperate classes as well.